skimage/util/dtype.py

```diff
@@ -431,3 +432,24 @@ def img_as_bool(image, force_copy=False):
 
     """
     return convert(image, np.bool_, force_copy)
+
+
+def img_as_float(image, force_copy=False):
```

Comment:
You can use the `@deprecated` decorator here instead of a makeshift warning. The advantage is that it allows you to specify the correct function to use instead.